codegen/src/lib.rs

```diff
@@ -67,20 +67,42 @@ pub fn deflate_utf8_file(ts: TokenStream) -> TokenStream {
 }
 
 fn inner(ts: TokenStream, utf8: bool) -> Result<impl Into<TokenStream>> {
+    if let Ok(t) = syn::parse::<ExprTuple>(ts.clone()) {
```

Comment:
Why tuple? The invocation will then become `deflate_file!(("file.gz", "base"))` with double parentheses.